src/lib/snark_params/pedersen.mli

```diff
@@ -28,29 +28,37 @@ module type S = sig
     type t = curve Quadruple.t array
   end
 
-  module Curve_chunk_table : sig
-    type t = {curve_points_table: curve array array}
-  end
-
   module State : sig
+    type chunk_table_fun = unit -> curve array array
+
     type t =
       { triples_consumed: int
       ; acc: curve
       ; params: Params.t
-      ; chunk_table: Curve_chunk_table.t }
+      ; get_chunk_table: chunk_table_fun }
 
     val create :
          ?triples_consumed:int
       -> ?init:curve
       -> Params.t
-      -> Curve_chunk_table.t
+      -> get_chunk_table:chunk_table_fun
       -> t
 
+    val update_fold_chunked : t -> bool Triple.t Fold.t -> t
+    (** use precomputed table of curve values *)
+
+    val update_fold_unchunked : t -> bool Triple.t Fold.t -> t
+    (** compute hash one triple at a time *)
+
     val update_fold : t -> bool Triple.t Fold.t -> t
+    (** dispatches to chunked or unchunked (default) version; called by daemon startup to use chunked version *)
```

Comment:
this "called by daemon ..." comment seems to be on the wrong item.

Reply:
Fixed.